web/impact/impact/v1/serializers/office_hours_serializer.py

```diff
@@ -32,6 +34,29 @@ class Meta:
             'topics', 'description', 'location',
         ]
 
+    def handle_conflicting_sessions(self, attrs):
+        office_hour = self.instance
+        start_time = attrs.get('start_date_time', None)
+        end_time = attrs.get('end_date_time', None)
+        skip_check = (office_hour and
+                      (office_hour.start_date_time == start_time and
+                       office_hour.end_date_time == end_time))
+        if (start_time or end_time) and not skip_check:
```

Comment:
Extracting the body of this conditional to a separate function may resolve the code climate warning